### PR TITLE
dev-python/pycollada: bump to 0.7.1

### PR DESCRIPTION
**Pull request by waebbl**
Additionally drop old 0.6.1

Closes: https://bugs.gentoo.org/717566
Package-Manager: Portage-2.3.99, Repoman-2.3.22
Signed-off-by: Bernd Waibel <waebbl@gmail.com>

### CODE REVIEW

File: dev-python/pycollada/pycollada-0.7.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+RDEPEND="
+	dev-python/lxml[${PYTHON_USEDEP}]
+	dev-python/numpy[${PYTHON_USEDEP}]
+	>=dev-python/python-dateutil-2.2[${PYTHON_USEDEP}]
+"
+DEPEND="
+	test? ( ${RDEPEND} )
+"
+BDEPEND="
+	dev-python/setuptools[${PYTHON_USEDEP}]
```
Comment (juippis) [inline dev-python/pycollada/pycollada-0.7.1.ebuild:29]: This is already handled by *distutils-r1.eclass*

File: dev-python/pycollada/pycollada-0.7.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	dev-python/lxml[${PYTHON_USEDEP}]
+	dev-python/numpy[${PYTHON_USEDEP}]
+	>=dev-python/python-dateutil-2.2[${PYTHON_USEDEP}]
+"
+DEPEND="
+	test? ( ${RDEPEND} )
+"
```
Comment (juippis) [inline dev-python/pycollada/pycollada-0.7.1.ebuild]: Could you try using the *distutils-r1.eclass* test function? It looks so simple in this ebuild I'm sure it'd work.

File: dev-python/pycollada/pycollada-0.7.1.ebuild [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+BDEPEND="
+	dev-python/setuptools[${PYTHON_USEDEP}]
+	doc? (
+		dev-python/sphinx[${PYTHON_USEDEP}]
+		media-gfx/graphviz
+	)
```
Comment (juippis) [inline dev-python/pycollada/pycollada-0.7.1.ebuild:33]: Same for doc building, can you attempt to use the eclass function?

File: dev-python/pycollada/pycollada-0.7.1.ebuild (hunk 2 above)
Comment (waebbl) [reply dev-python/pycollada/pycollada-0.7.1.ebuild]: The eclass function works. Thanks :)

File: dev-python/pycollada/pycollada-0.7.1.ebuild (hunk 3 above)
Comment (waebbl) [reply dev-python/pycollada/pycollada-0.7.1.ebuild:33]: Same here.

**Pull request closed without merging**